pycyphal/transport/udp/_ip/__init__.py

```diff
@@ -5,15 +5,14 @@
 from ._socket_factory import SocketFactory as SocketFactory
 from ._socket_factory import Sniffer as Sniffer
 
-from ._endpoint_mapping import IP_ADDRESS_NODE_ID_MASK as IP_ADDRESS_NODE_ID_MASK
-from ._endpoint_mapping import MULTICAST_GROUP_SUBJECT_ID_MASK as MULTICAST_GROUP_SUBJECT_ID_MASK
-from ._endpoint_mapping import SUBJECT_PORT as SUBJECT_PORT
-from ._endpoint_mapping import node_id_to_unicast_ip as node_id_to_unicast_ip
-from ._endpoint_mapping import unicast_ip_to_node_id as unicast_ip_to_node_id
+from ._v4 import IPv4SocketFactory as IPv4SocketFactory
```

Comment:
As far as I see this is not actually needed. The only place of use I found is inside UDPTransport and it is incorrect (more on this later).